core/server/common/src/main/java/alluxio/web/UIFileInfo.java

```diff
@@ -109,8 +109,8 @@ public LocalFileInfo(String name, String absolutePath, long size, long creationT
   public UIFileInfo(URIStatus status) {
     // detect the extended acls
     AccessControlList acl = AccessControlList
-        .fromStringEntries(status.getOwner(), status.getGroup(), status.getAclEntries());
-    boolean hasExtended = acl.hasExtended() || !status.getDefaultAclEntries().isEmpty();
+        .fromStringEntries(status.getOwner(), status.getGroup(), status.getAcl().toStringEntries());
```

Comment:
is there no ACL copy constructor?

Reply:
I can just use `status.getAcl()` directly.